### PR TITLE
Unified text outputs

### PR DESCRIPTION
**Pull request by GChuf**
A few very minor changes to the text outputs.
Includes a few dots and spaces changes.

### CODE REVIEW

File: src/bitcoin-cli.cpp [hunk 1]
```diff
@@ -44,14 +44,14 @@ static void SetupCliArgs()
...
+    gArgs.AddArg("-conf=<file>", strprintf("Specify configuration file. Relative paths will be prefixed by datadir location (default: %s).", BITCOIN_CONF_FILENAME), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-datadir=<dir>", "Specify data directory", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
-    gArgs.AddArg("-getinfo", "Get general information from the remote server. Note that unlike server-side RPC calls, the results of -getinfo is the result of multiple non-atomic requests. Some entries in the result may represent results from different states (e.g. wallet balance may be as of a different block from the chain state reported)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-getinfo", "Get general information from the remote server. Note that unlike server-side RPC calls, the result of -getinfo is the result of multiple non-atomic requests. Some entries in the result may represent results from different states (e.g. wallet balance may be as of a different block from the chain state reported).", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```
Comment (promag) [inline src/bitcoin-cli.cpp]: +0 on result vs results.

**Review by promag — COMMENTED** [review]
NACK.

FYI the PR implementing https://github.com/bitcoin/bitcoin/pull/15493#issuecomment-473727102 will update the lines with `gArgs.AddArg`.

I don't see value in changing the dot position. The same for the new whitespaces.

File: src/bitcoin-cli.cpp (hunk 1 above)
Comment (gapeman) [reply src/bitcoin-cli.cpp]: It should either be "the result ... is" or "the results ... are".

**Comment by gapeman** [comment]
@promag just to make sure, I have deleted the 2 unnecessary spaces, not added them.
The dots positions should always be at the end of a sentence - so after all the brackets.
I changed the positions so that everything is written in the same style.

I also think "results ... is" is a grammatical mistake, that's why I changed it to "result ... is".

All of the above combined adds no value to the code, but the way the messages are written matters as well. I am pleased that people put effort in nice messages and correct grammar in this project, so I wanted to make a small contribution myself. Details are important as well.

Please reconsider.

**Comment by fanquake** [comment]
Thanks, however the project is overwhelmed with PRs, and is focusing on those with a higher-impact.

Please consider looking at the [good first issue](https://github.com/bitcoin/bitcoin/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) list if you are looking for ways to contribute.

If you are considering further trivial PRs, please at least make sure they are immediately mergeable. i.e use a more descriptive commit `"docs: unified argument descriptions in bitcoin-cli.cpp"`.

**Pull request closed without merging**